py_inspect.py

```diff
@@ -1064,8 +1123,18 @@ def __generate_props_dict(self, element_info):
             ['top_level_parent', str(element_info.top_level_parent)]
         ] if (self.backend == 'uia') else []
 
+        props_wpf = [
+            ['auto_id', str(element_info.auto_id)],
```

Comment:
Not all properties are present. At least `"name"` and `"value"` must be here. It's even better to get full list of properties from static members of ElementInfo based class. But using static members is more global task related to all backends. I would leave this task to @KimNikita.